modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/rich/text/RichTextDDMFormFieldTemplateContextContributor.java

```diff
@@ -75,7 +75,8 @@
 
 		EditorConfiguration editorConfiguration =
 			EditorConfigurationFactoryUtil.getEditorConfiguration(
-				StringPool.BLANK, ddmFormFieldType, "ckeditor_classic",
+				JournalPortletKeys.JOURNAL, ddmFormFieldType,
```

Comment:
Hey @fortunatomaldonado, why did you made this change? This could impact other products that use RichText, making them use just like Journal.

Reply:
Hello @mateussgn,

I changed it to `JournalPOrtletKeys.Journal` since I had to include the [`javax.portlet.name`](https://github.com/liferay-data-engine/liferay-portal/pull/563/files#diff-8795a265b9cd40cf690030254bf9ad13031b06fa56deb807f54187d6008659acR35) in order to use the new ConfigContributor.

I found that in [`BaseEditorProvider`](https://github.com/liferay-data-engine/liferay-portal/pull/563/files#diff-8795a265b9cd40cf690030254bf9ad13031b06fa56deb807f54187d6008659acR35) it tries to match the `portletName, editorConfigKey, editorName` and in the [`_matches`](https://github.com/liferay/liferay-portal-ee/blob/master/portal-impl/src/com/liferay/portal/editor/configuration/BaseEditorProvider.java#L146) it will return true is the string is empty or matches. I figured that since it was previously `StringPool.BLANK`, this would not make any change.

Let me know if my assumption is misguided or if anything needs to be changed.
Thank you for taking a look at this!

Reply:
Nice @fortunatomaldonado, 

I was observing, and instead of pass the `JournalPortletKeys.JOURNAL` directly, would be better to do it dynamically, since the `RichTextDDMFormFieldTemplateContextContributor.java` could be used by other portlets.

It is possible to get the `portletName` on the `ddmFormFieldRenderingContext` argument, it will have the `_portletNamespace` which is very similar to the `portletName`. The difference between them is that the `_portletNamespace` has underlines at the beginning and at the end of the string.

Could you please make these changes?

Given what I said, I'll close this pull request.

Reply:
Hello @mateussgn,

Thank you for reviewing this.
I am a bit confused on what exactly the change you are requesting. Can you elaborate more?